server/tests/audit_math/test_ballot_polling.py

```diff
@@ -0,0 +1,91 @@
+# pylint: disable=invalid-name
+from decimal import Decimal
+import pytest
+
+from ...audit_math import bravo, minerva, ballot_polling
+from ...audit_math.sampler_contest import Contest
+from ...models import AuditMathType
+
+SEED = "12345678901234567890abcdefghijklmnopqrstuvwxyz😊"
+RISK_LIMIT = 10
+ALPHA = Decimal(0.1)
+
+
+@pytest.fixture
+def contests():
+    contests = {}
+
+    for contest in bravo_contests:
+        contests[contest] = Contest(contest, bravo_contests[contest])
+
+    return contests
+
+
+def test_get_sample_sizes(contests):
+    for contest in contests:
+        bravo_sample_size = bravo.get_sample_size(
+            RISK_LIMIT, contests[contest], None, None
+        )
+
+        assert bravo_sample_size == ballot_polling.get_sample_size(
+            RISK_LIMIT, contests[contest], None, AuditMathType.BRAVO, None
+        )
+
+        minerva_sample_size = minerva.get_sample_size(
+            RISK_LIMIT, contests[contest], None, None
+        )
+        assert minerva_sample_size == ballot_polling.get_sample_size(
+            RISK_LIMIT, contests[contest], None, AuditMathType.MINERVA, None
+        )
+
+
+def test_compute_risk(contests):
+    print("testing!")
+    for contest in contests:
+        print(contest)
+        sample = round1_sample_results[contest]
+
+        bravo_test_stat, bravo_decision = bravo.compute_risk(
+            RISK_LIMIT, contests[contest], sample
+        )
+
+        computed_stat, computed_decision = ballot_polling.compute_risk(
+            RISK_LIMIT, contests[contest], sample, AuditMathType.BRAVO, {1: 119}
+        )
```

Comment:
I would do this kind of test differently, just ensuring that `ballot_polling.compute_risk(...)` calls the proper `bravo.compute_risk`, as this test here could potentially be pretty inefficient... but I can live with it, just a comment.